.migration-backup/ui-components/src/ui/atoms/BackgroundPattern/BackgroundPattern.tsx

```
@@ -0,0 +1,335 @@
+
+// Error boundary component for SVG rendering
+class SvgErrorBoundary extends React.Component<{
+  children: React.ReactNode;
+  fallback?: React.ReactNode;
+}, {hasError: boolean;}> {
+  constructor(props: {children: React.ReactNode;fallback?: React.ReactNode;}) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError(error: Error): {hasError: boolean;} {
```

ESLint

[error] 'error' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)